src/ccl/charts/chart.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, ViewEncapsulation, Inject } from '@angular/core';
+import { ChartService } from '../services/chart.service';
+import { LineGraph } from './line-graph.component';
+
+/*
+ * Chart component
+ * Management for the sections that appear in the chart container
+ */
+
+@Component({
+  selector: 'charts',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
You're right this method was not correct here. default emulated is fine. All the other components seem to require the None method. I don't think I understand Shadow DOM well even after reading though: http://blog.thoughtram.io/angular/2015/06/29/shadow-dom-strategies-in-angular2.html